l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -72,6 +77,8 @@ def _domain_ir_values(self):
         comodel_name='ir.values',
         domain=_domain_ir_values,
         help='This report will be automatically included in the created XML')
+    ignore_errors = fields.Boolean(
+        string="Export anyway ignoring errors.")
```

Comment:
@sergiocorato potresti spiegare meglio? Se ho ben capito, le fatture con errori non vengono esportate, ma saltate, giusto?

Reply:
esatto, così l'utente poi passa una per una solo quelle che non vanno

Reply:
Allora il campo lo chiamerei ad esempio `Skip e-invoices with errors` e aggiungerei un help tipo `Do not block massive export when some e-invoices raise error. Just skip them`

Reply:
fatto, grazie